.github/workflows/continous-integration.yml

```diff
@@ -540,15 +540,26 @@ jobs:
           coverage combine "${final_dir}/"*
           coverage xml
 
-      - name: Upload reports to codeclimate
+  sonarcloud:
+    name: SonarCloud Scan
+    if: github.ref_type != 'tag'
+    runs-on: ubuntu-22.04
+    needs:
+      - upload_coverage_reports
```

Comment:
Did you forget to update the job name here?

Reply:
Yup i did, thanks:)